full/Dockerfile

```diff
@@ -228,6 +233,7 @@ RUN curl -sSL https://rvm.io/mpapis.asc | gpg --import - \
         rvm requirements \
         && rvm install 2.5 \
```

Comment:
If we are adding 2.7 let's remove 2.5 for the sake of image size

Reply:
https://www.jetbrains.com/lp/devecosystem-2019/ruby/

Reply:
Fair enough

Reply:
@jankeromnes is it ok adding an additional Ruby version?

Reply:
You know what, I'm just going to remove some useless stuff from the image hang on

Reply:
Ok so the image should be much smaller now (the logs too, I fixed an issue where it was more than 40000 lines long for no real purpose)